test/common/autoscaling.go

```diff
@@ -0,0 +1,98 @@
+package common
+
+import (
+	"context"
+	rhmiv1alpha1 "github.com/integr8ly/integreatly-operator/apis/v1alpha1"
+	"github.com/integr8ly/integreatly-operator/pkg/addon"
+	k8sclient "sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+	"time"
+)
+
+const (
+	hpaQuotaParam = "500"
+	hpaQuotaName  = "50 Million"
+)
+
+func TestAutoscaling(t TestingTB, ctx *TestingContext) {
+	quotaConfig, err := getQuotaConfig(t, ctx.Client)
+	if err != nil {
+		t.Fatalf("Error retrieving Quota: %v", err)
+	}
+
+	installation, err := GetRHMI(ctx.Client, true)
+	if err != nil {
+		t.Fatal("couldn't get RHMI cr for quota test")
+	}
+	if installation == nil {
+		t.Fatalf("Got invalid rhmi CR: %v", installation)
+	}
+	initialHPAValue := installation.Spec.AutoscalingEnabled
+
+	// wait if stage no complete
+	startTime := time.Now()
+	endTime := startTime.Add(time.Minute * time.Duration(timeoutWaitingQuotachange))
+
+	for startTime.Before(endTime) {
+		startTime = time.Now()
+		if installation.Status.Stage == rhmiv1alpha1.CompleteStage {
+			break
+		}
+	}
+
+	//verify that the TOQuota value is set and that Quota is not set
+	//assuming this is run after installation
+	if installation.Status.Quota == "" {
+		t.Fatal("Expected '.status.quota' to contain a quota name after installation, but got an empty string")
+	}
+	if installation.Status.ToQuota != "" {
```

Comment:
I'm not sure why are we checking this. 
In general, before this test will run you are guaranteed to have the following:
1. Installation fully completed
2. Therefore, quota present and toQuota not present.